client/packages/common/src/api/GqlContext.tsx

```diff
@@ -117,12 +117,13 @@ class GQLClient extends GraphQLClient {
     // without it, the page will re-render continuously
     return response.then(
       data => (data ?? this.emptyData) as T,
-      ({ response }) => {
+      reason => {
```

Comment:
offroad: was a suggestion in [this comment](https://github.com/openmsupply/open-msupply/pull/1677#pullrequestreview-1413945767) which I thought would be helpful so have included it

Reply:
Just wondering did you manage to replicate and test this ?

Reply:
I can't remember exactly how unfortunately! I did something like run the release build of the server with self-signed certs and connect with the IP of the server to get a CORS error. So I didn't replicate the specific issue - but a general problem with the graphQL context

Reply:
Oki, not sure how to validate this code without test, it looks safe, but can't validate if it fixes the comment